appendix_abstractions.asciidoc

```diff
@@ -25,18 +25,48 @@ change our systems.
 We can reduce the degree of coupling within a system by abstracting away the
 details.
 
-[[TODO: foo the poo]]
 
-[[TODO: Do the macGroo]]
+[[coupling_illustration1]]
+.Lots of coupling
+====
+[source,python]
+----
++--------+     +--------+
+|        |<--->|        |
+|        |<--->|        |
+| System |<--->| System |
+|   A    |<--->|   B    |
+|        |<--->|        |
+|        |<--->|        |
++--------+     +--------+
+----
+====
+
+[[coupling_illustration2]]
+.Less coupling
+====
+[source,python]
+----
++--------+     +--------+     +--------+
+|        |<--->|        |     |        |
+|        |<--->|        |     |        |
+| System |<--->|  Abs-  |     | System |
+|   A    |<--->|traction|<--->|   B    |
+|        |<--->|        |     |        |
+|        |<--->|        |     |        |
++--------+     +--------+     +--------+
+----
+====
+
 
 In both diagrams above, we have a pair of subsystems, with the one dependent on
-the other. In the first diagram, there is a high degree of coupling betweeen the
-two becausages of sausages. If we need to change feature B, there's a good
-chance that the change will ripple through to feature A.
+the other. In the first diagram, there is a high degree of coupling between the
+two because of reasons. If we need to change system B, there's a good
```

Comment:
This passage was deliberately nonsensical so that we'd have to come back and think of a proper example. Is this proposed final text?

Reply:
it's proposed MVP text lol